src/main/java/it/gov/pagopa/payhub/activities/service/ingestionflow/organization/OrganizationProcessingService.java

```diff
@@ -0,0 +1,105 @@
+import it.gov.pagopa.payhub.activities.service.ingestionflow.IngestionFlowProcessingService;
+import it.gov.pagopa.pu.organization.dto.generated.Broker;
+import it.gov.pagopa.pu.organization.dto.generated.Organization;
+import it.gov.pagopa.pu.processexecutions.dto.generated.IngestionFlowFile;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+@Service
+@Lazy
+@Slf4j
+public class OrganizationProcessingService extends IngestionFlowProcessingService<OrganizationIngestionFlowFileDTO, OrganizationIngestionFlowFileResult, OrganizationErrorDTO> {
+
+    private final OrganizationMapper organizationMapper;
+    private final OrganizationService organizationService;
+
+
+    public OrganizationProcessingService(
+            OrganizationMapper organizationMapper,
+            OrganizationErrorsArchiverService organizationErrorsArchiverService,
+        OrganizationService organizationService) {
+        super(organizationErrorsArchiverService);
+        this.organizationMapper = organizationMapper;
+        this.organizationService = organizationService;
+    }
+
+
+    public OrganizationIngestionFlowFileResult processOrganization(
+            Iterator<OrganizationIngestionFlowFileDTO> iterator,
+            List<CsvException> readerException,
+            IngestionFlowFile ingestionFlowFile, Path workingDirectory) {
+        List<OrganizationErrorDTO> errorList = new ArrayList<>();
+        OrganizationIngestionFlowFileResult ingestionFlowFileResult = new OrganizationIngestionFlowFileResult();
+        ingestionFlowFileResult.setOrganizationIpaCodeList(new ArrayList<>());
+
+        process(iterator, readerException, ingestionFlowFileResult, ingestionFlowFile, errorList, workingDirectory);
+        return ingestionFlowFileResult;
+    }
+
+    @Override
+    protected boolean consumeRow(long lineNumber, OrganizationIngestionFlowFileDTO organizationDTO, OrganizationIngestionFlowFileResult ingestionFlowFileResult, List<OrganizationErrorDTO> errorList, IngestionFlowFile ingestionFlowFile) {
+        try {
+            Optional<Organization> organizationBroker = organizationService.getOrganizationById(ingestionFlowFile.getOrganizationId());
+            Long brokerId = organizationBroker.map(Organization::getBrokerId).orElse(null);
+
+            if (brokerId == null) {
+                log.error("Broker with fiscal code {} not found", organizationDTO.getBrokerCf());
+                OrganizationErrorDTO error = new OrganizationErrorDTO(
+                        ingestionFlowFile.getFileName(), organizationDTO.getIpaCode(),
+                        lineNumber, "BROKER_NOT_FOUND", "Broker not found");
+                errorList.add(error);
+                return false;
+            }
```

Comment:
let's move these line before calling the `process` method into `processOrganization` method.
if there is not broker, let's return an `OrganizationIngestionFlowFileResult` the error description describe here: this is a global error, not a row  dependant error.

Let's add to OrganizationIngestionFlowFileResult the fields brokerId and brokerFiscalCode, setting them before calling the `process` method if the previous check was successful.